less/helpers.less

```diff
@@ -331,7 +331,11 @@ p {
 .center-items-align {
   align-items: center;
 }
-
+.associated-documents {
+  .quality {
+    display: none;
```

Comment:
Is it to avoid the position conflict with the "remove association" icon in the bottom right corner of cards?
Does it mean the quality is only shown in the advanced search pages?